medusa/helper/common.py

```diff
@@ -310,7 +296,7 @@ def try_int(candidate, default_value=0):
         return int(candidate)
     except (ValueError, TypeError):
         if candidate and ("," in candidate or "." in candidate):
-            logger.error(u"Failed parsing provider. Traceback: %r" % traceback.format_exc())
+            log.error(u"Failed parsing provider. Traceback: %r" % traceback.format_exc())
```

Comment:
Are we going to accept non-lazy logging? I think this should then at least be changed to: 
`log.error(u"Failed parsing provider. Traceback: %r", traceback.format_exc())`